app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -0,0 +1,276 @@
+# -*- encoding : utf-8 -*-
```

Comment:
It's not overriding anything but is required for the Bookmark model to work (specifically the bookmarked? document) method.